src/components/portrait/portrait.component.js

```diff
@@ -6,11 +6,57 @@ import PortraitGravatar from './portrait-gravatar.component';
 import PortraitInitials from './portrait-initials.component';
 import StyledPortrait, { StyledCustomImg, StyledIcon } from './portrait.style';
 
-function Portrait(props) {
+function Icon(iconProps) {
+  const { darkBackground, shape, size } = iconProps;
+  return (
+    <StyledIcon
+      type='individual'
+      size={ size }
+      shape={ shape }
+      darkBackground={ darkBackground }
+    />
+  );
+}
+
+function Initials(initialsProps) {
   const {
-    alt, darkBackground, gravatar, initials, shape, size, src
-  } = props;
+    alt, darkBackground, initials, size
+  } = initialsProps;
+  return (
+    <PortraitInitials
+      size={ size }
+      initials={ initials }
+      darkBackground={ darkBackground }
+      alt={ alt }
+    />
+  );
+}
+
+function Gravatar(gravatarProps) {
+  const { alt, gravatar, size } = gravatarProps;
+  return (
+    <PortraitGravatar
+      gravatarEmail={ gravatar }
+      size={ size }
+      alt={ alt }
+    />
+  );
+}
+
+function CustomImg(customimgProps) {
+  const { alt, size, src } = customimgProps;
+  return (
+    <StyledCustomImg
+      src={ src }
+      alt={ alt }
+      size={ size }
+      data-element='user-image'
+    />
+  );
+}
```

Comment:
adding these elements as components could create confusion. What I had in mind:
```
function renderGravatar() {
  if(!gravatar) return null;

  return  (
    <PortraitGravatar
      gravatarEmail={ gravatar }
      size={ size }
      alt={ alt }
     />
  );
}
```

Reply:
Ah OK, I see what you mean, thanks. Refactored in da572ce :thumbsup: